src/ProjectTemplates/Web.ProjectTemplates/content/RazorComponentsWeb-CSharp/Components/Pages/Counter.razor

```diff
@@ -1,10 +1,10 @@
-﻿@page "/counter"
+@page "/counter"
 
 <h1>Counter</h1>
 
 <p>Current count: @currentCount</p>
 
-<button class="btn btn-primary" onclick="@IncrementCount">Click me</button>
+<button class="btn btn-primary" onclick="@((Action)IncrementCount)">Click me</button>
```

Comment:
@rynowak Without the cast, this gives an error about ambiguous method match in VS, even though it compiles OK on the command line. This problem will go away with https://github.com/aspnet/AspNetCore-Tooling/pull/222, will it?

Reply:
What are the different cases of the ambiguity? This is a serious bug if it's not fixed by the compiler change.

Reply:
This is the error when building in VS using the 3.0.100-preview4-010380 SDK:

```
1>obj\Debug\netcoreapp3.0\Razor\Components\Pages\Counter.razor.g.cs(34,92,34,162): error CS0121: The call is ambiguous between the following methods or properties: 'BindMethods.GetEventHandlerValue<T>(Action)' and 'BindMethods.GetEventHandlerValue<T>(Func<Task>)'
```

Reply:
I can't think of how anything we've done would cause or fix this. We have always had these two methods defined - and we definitely test cases like this. 

Is the language version set to 7.3 for the project?

Reply:
Ah yes, it was `LangVersion`! Once that's in, the problem goes away and the workaround isn't needed. Thanks for the pointer.